content/docs/gto/user-guide/index.md

```diff
@@ -1,122 +1,220 @@
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+jobs:
+  act:
+    name: Deploy a model upon stage assignment
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - id: gto
+        uses: iterative/gto-action@v2
+      - uses: actions/setup-python@v2
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip setuptools wheel
+          pip install -r requirements.txt
+      - if: steps.gto.outputs.event == 'assignment'
+        run: |
+          # TODO: check this works
+          mlem deployment run \
+            --load deploy/${{ steps.gto.outputs.stage }} \
+            --model ${{ steps.gto.outputs.path }}
 ```
 
-## Acting in CI/CD
+This relies on having [deployment declarations](/doc/user-guide/deploying) in
+the `deploy/` directory, such as:
 
-A popular deployment option is to use CI/CD (triggered when Git tags are
-pushed). For general details, check out something like
-[GitHub Actions](https://github.com/features/actions),
-[GitLab CI/CD](https://docs.gitlab.com/ee/ci/) or
-[Circle CI](https://circleci.com).
+```yaml
+# deploy/dev.yaml
+object_type: deployment
+type: heroku
+app_name: mlem-dev
+```
 
-The other option is to
-[configure webhooks](https://docs.github.com/en/rest/webhooks) that will send
-HTTP requests to your server upon pushing Git tags to the remote.
+This declaration is read by MLEM in CI and the model promoted to `dev` is
+deployed to https://mlem-dev.herokuapp.com.
 
-Finally, you can configure your server to query your Git provider via something
-like REST API to check if changes happened. As an example, check out
-[Github REST API](https://docs.github.com/en/rest).
+Note, that you need to provide environment variables to deploy to Heroku and
+update the [deployment state](/doc/user-guide/deploying). The location for the
+state should be
+[configured](/doc/user-guide/deploying#setting-up-remote-state-manager) in MLEM
+config file:
+
+```yaml
+# .mlem.yaml
+core:
+  state:
+    uri: s3://bucket/path
+```
 
-### Getting started with CI/CD
+Check out [another example](https://github.com/iterative/example-gto/tree/mlem)
+of MLEM model deployment in the `main` branch of the `example-gto` repo.
 
-To act upon registrations and assignments (Git tags), you can create a simple CI
-workflow. To see an example, check out
-[the workflow in `example-gto` repo](https://github.com/iterative/example-gto/blob/main/.github/workflows/gto-act-on-tags.yml).
-The workflow uses [the GTO GH Action](https://github.com/iterative/gto-action)
-that fetches all Git tags (to correctly interpret the Registry), finds out the
-`version` of the artifact that was registered, the `stage` that was assigned,
-and annotations details such as `path`, `type`, `description`, etc, so you could
-use them in the next steps of the CI.
+</tab>
+</toggle>
 
-### Helpful commands
+## Using DVC to annotate artifacts
```

Comment:
As mentioned above, I think we should drop this section since it's not really about GTO.